README.md

```diff
@@ -1,32 +1,37 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<!-- about  Microverse first project -->
 
-# Microver Capstone 1 
+ # Microverse First project  
+<img src="assert/desktop-version-screenshot.png">
 
-> Description the project.
+> Description of  the project.
 
+ This is a portfolio website 
 
 ## Built With
 
-- HTML,CSS
-- JAVASCRIPT
-- GIT
+- Html
+- Css
+- Github flow 
 
```

Comment:
- [ ] kindly be noticed that you should record a video presenting the features of the project.
- [ ] The maximum length of the video is 5 minutes.